config/application.rb

```diff
@@ -48,11 +48,6 @@ class Application < Rails::Application
     # logging use the Logger::Formatter.new.
     config.log_formatter = JsonLogFormatter.new
 
-    if ENV["RAILS_LOG_TO_STDOUT"].present?
-      config.logger = ActiveSupport::Logger.new($stdout)
-      config.logger.formatter = config.log_formatter
-    end
-
```

Comment:
If I'm reading this right, I think this change will mean we will never log to stdout, I think we want to always log to stdout though?

Reply:
Sorry should have referenced here: https://github.com/alphagov/forms-product-page/pull/88/files#diff-da60b4e96eff2b132991226d308949e23f4ef3aad45ad59edd09cbc32cc6251eR41